packages/app/src/state/exchange/actions.ts

```diff
@@ -466,6 +467,13 @@ export const setRatio =
 		}
 	}
 
+export const fetchWalletTrades = createAsyncThunk<SynthExchange[], void, ThunkConfig>(
+	'exchange/fetchWalletTrades',
+	(_, { extra: { sdk } }) => {
+		return sdk.exchange.getWalletTrades()
```

Comment:
try catch and notify on error would be nice

Reply:
But if we delete sport history then we could just delete this

Reply:
Yeah could just remove Spot History table, it's part of #2191

Reply:
Removed.